src/bin/purge_db.rs

```diff
@@ -1,181 +0,0 @@
-use anyhow::Result;
```

Comment:
It's not obvious to me that this is worth removing rather than transforming it to work with `sqlite`? I guess ultimately, we're not currently using this script in devops. But at some point I imagine we will? But maybe that's a big effort, and not really worth tackling at this point.

Reply:
I thought about this but we still don't know what exactly our database would look like when we refactor it to support beacon and state. I think it is more productive to remove it now and rewrite it if needed at a later stage.